services/tenant-ui/frontend/src/components/reservation/Status.vue

```diff
@@ -61,10 +61,10 @@
   </form>
 
   <!-- Statuses to check -->
-  <div v-if="loading" class="flex justify-content-center">
+  <div v-if="loading" class="flex justify-content-center status-spinner">
     <ProgressSpinner />
   </div>
-  <div v-else>
```

Comment:
We wouldn't want to remove this v-else I think. If loading the status pane below should not be there (albeit each one v-if'd themselves but in case anything else needs to be added)

Reply:
Yeah need to keep the v-else:

![image](https://user-images.githubusercontent.com/17445138/228085436-741446fa-918b-4c7d-bf3d-30824d5c3cd2.png)

Reply:
> Yeah need to keep the v-else (or do some loading handling on the validate form)
> 
> ![image](https://user-images.githubusercontent.com/17445138/228085436-741446fa-918b-4c7d-bf3d-30824d5c3cd2.png)

Using the v-else breaks what I'm trying to do. We can maybe restyle it.

Reply:
That shouldn't break it at all. What's happening?

Reply:
It may make more sense to move the loading widget into whatever button was pressed. Which I think our UI framework can handle.

Reply:
Yeah when validating each of the forms it's probably better to do loaders and disablers on their buttons themselves. Can add that in later.

Reply:
Yeah that'll be better then I think (can disable the textboxes conditionally as well if that applies)